data/anaconda.conf

```diff
@@ -158,6 +158,13 @@ preserved_arguments =
     biosdevname ipv6.disable net.ifnames net.ifnames.prefix
     nosmt
 
+# Disable the GRUB submenu
+disable_submenu = True
+
+# Set GRUB terminal type
+terminal_type =
```

Comment:
Same here, especially if it uses some GRUB specific terminal type identifiers apparently, then I think it should be namespaced to grub.